### ISSUE TITLE
About Geometric Loss

### ISSUE THREAD

**Issue by dontLoveBugs** [youngguncho/PoseNet-Pytorch#2]
Do you implement geometric loss function?

**Comment by youngguncho** [comment]
Not yet. Currently,  I implemented only posenet and bayesian posenet.

**Comment by dontLoveBugs** [comment]
Thanks
